hwdb.d/60-keyboard.hwdb

```diff
@@ -511,6 +511,8 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pn*:*
  KEYBOARD_KEY_92=brightnessdown                         # Fn+F7 (Fn+F9 on 6730b)
  KEYBOARD_KEY_97=brightnessup                           # Fn+F8 (Fn+F10 on 6730b)
  KEYBOARD_KEY_ee=switchvideomode                        # Fn+F4
+ KEYBOARD_KEY_66=pickup                                 # Pickup phone button
+ KEYBOARD_KEY_65=hangup                                 # Hangup phone button
```

Comment:
> Keycode KEY_PICKUP unknown
> Keycode KEY_HANGUP unknown

@keszybz @hadess how are these values defined?

Reply:
Apparently from python-evdev, as per https://github.com/systemd/systemd/blob/main/hwdb.d/parse_hwdb.py#L42
But I read the sources, so 🤷